### PR TITLE
Update kubespawner ldapauthenticator

### PR DESCRIPTION
**Pull request by manics**
This should bring in kubespawner 7.0.0 and ldapauthenticator 2.0.1

### CODE REVIEW

File: images/hub/requirements.in [hunk 1]
```diff
@@ -12,14 +12,14 @@ jupyterhub==5.2.1
...
-jupyterhub-ldapauthenticator==2.0.0
+jupyterhub-ldapauthenticator>=2.0.1
 jupyterhub-ltiauthenticator!=1.3.0
 jupyterhub-nativeauthenticator
 jupyterhub-tmpauthenticator
 oauthenticator[googlegroups,mediawiki]
 
 ## Kubernetes spawner
-jupyterhub-kubespawner==7.0.0b3
+jupyterhub-kubespawner>=7.0.0
```
Comment (minrk) [inline images/hub/requirements.in:22]:
```suggestion
jupyterhub-kubespawner==7.*
```

Comment (minrk) [reply images/hub/requirements.in:22]: maybe keep the major pin on kubespawner?

Comment (consideRatio) [reply images/hub/requirements.in:22]: I figure we should either not have pins or systemticallcy pin major versions, fine with either. Do you see a reason to pin just kubespawner Min?

Comment (minrk) [reply images/hub/requirements.in:22]: I'm fine removing the pins. It's all pinned in requirements.txt, anyway.

kubespawner is special in that much of the chart itself is deeply tied to KubeSpawner in many ways. That's not really true of any package other than kubespawner and jupyterhub, so it doesn't seem odd to me to have special treatment of these two packages in that way.

I'm happy with this PR as-is, then.

**Pull request merged**